src/app/configs/data/shortcuts.xml

```diff
@@ -352,7 +352,7 @@
     </SC>
   <SC>
     <key>stretch+</key>
-    <seq>}</seq>
+    <seq>}</seq><!-- same as 'Ctrl+Alt+0' for 'voice-assignment-all-in-instrument' on German keyboards! -->
```

Comment:
Although with the advent of system locks in 4.5, the stretch shortcuts lost a lot of their importance...